buildpack.md

```diff
@@ -397,13 +419,15 @@ The purpose of build is to transform application source code into runnable artif
 During the build phase, typical buildpacks might:
 
 1. Read the Buildpack Plan in `<plan>` to determine what dependencies to provide.
-2. Provide the application with dependencies for launch in `<layers>/<layer>`.
-3. Provide subsequent buildpacks with dependencies in `<layers>/<layer>`.
-4. Compile the application source code into object code.
-5. Remove application source code that is not necessary for launch.
-6. Provide start command in `<layers>/launch.toml`.
-7. Write a partial Bill-of-Material to `<layers>/launch.toml` describing any provided application dependencies.
-8. Write a partial Bill-of-Material to `<layers>/build.toml` describing any provided build dependencies.
+1. Provide the application with dependencies for launch in `<layers>/<layer>`.
+1. Reuse application dependencies from a previous image by appending by `launch = true` to `<layers>/<layer>.toml`.
```

Comment:
```suggestion
1. Reuse application dependencies from a previous image by appending `types.launch = true` to `<layers>/<layer>.toml`.
```

Reply:
@natalieparellano, I thought that you, @ekcasey and I decided to keep using `launch`, `build` and `cache` and not change them to `types.launch`, `types.build` and `types.cache` for clarity purposes.
If we decide to add `types`, there are more places in this file that should be changed.

Reply:
Summarizing side conversation - we think that when talking about "layers" marked `cache = true` we don't need `types`, but we should give accurate instructions for modifying the file.

Reply:
I think we should add a sentence or two at the beginning of the `Layer Types` section that explains these keys are under the `[types]` table, and then we can refer to them only as `launch`, `build`, `cache` from there on (but only within the `Layer Types` section. If there are uses outside of that section, we should reevaluate.

Reply:
Please see https://github.com/buildpacks/spec/pull/209/commits/81cb981a51b6186238559a28a327366160fa2024.